recipe_sha: 95d2069bf58001b60b722e688e9ac9e323598780d078923f611d84e1af582e94  # The commit at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |-
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.3s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.4s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.5s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.6s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.7s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.8s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.9s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 8.0s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 8.1s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 8.2s
  conda-forge/linux-64 100%[2K[1A[2K[0Gconda-forge/linux-64
  [?25h[34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0G[] 0.0s
  [2K[1A[2K[0Gopt/conda/conda-bld/linux-64                        ??.?MB @  ??.?MB/s 0 failed  0.0s
  opt/conda/conda-bld/noarch                         127.0 B @   2.8MB/s  0.0s
  [?25hMamba failed to solve:
   - cmake
   - make
   - gfortran_linux-64 12.*
   - gcc_linux-64 10.*
   - gxx_linux-64 10.*

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - package gfortran_linux-64-12.1.0-h7ec3724_10 requires gcc_linux-64 12.1.0 h3bb4806_10, but none of the providers can be installed
  -

  Leaving build/test directories:
    Work:
   /opt/conda/conda-bld/work
    Test:
   /opt/conda/conda-bld/test_tmp
  Leaving build/test environments:
    Test:
  source activate  /opt/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
    Build:
  source activate  /opt/conda/conda-bld/_build_env


  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 141, in mamba_get_install_actions
      solution = solver.solve_for_action(_specs, prefix)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 230, in solve_for_action
      t = self.solve(specs)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 220, in solve
      raise RuntimeError("Solver could not find solution."  error_string)
  RuntimeError: Solver could not find solution.Mamba failed to solve:
   - cmake
   - make
   - gfortran_linux-64 12.*
   - gcc_linux-64 10.*
   - gxx_linux-64 10.*

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - package gfortran_linux-64-12.1.0-h7ec3724_10 requires gcc_linux-64 12.1.0 h3bb4806_10, but none of the providers can be installed
  -

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2038, in build
      output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 787, in expand_outputs
      for (output_dict, m) in deepcopy(_m).get_output_metadata_set(permit_unsatisfiable_variants=False):
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 2524, in get_output_metadata_set
      conda_packages = finalize_outputs_pass(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 884, in finalize_outputs_pass
      fm = finalize_metadata(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 547, in finalize_metadata
      build_unsat, host_unsat = add_upstream_pins(m,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 387, in add_upstream_pins
      build_deps, build_unsat, extra_run_specs_from_build = _read_upstream_pin_files(m, 'build',
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 374, in _read_upstream_pin_files
      deps, actions, unsat = get_env_dependencies(m, env, m.config.variant,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 131, in get_env_dependencies
      actions = environ.get_install_actions(tmpdir, tuple(dependencies), env,
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 150, in mamba_get_install_actions
      raise err
  conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("gcc_linux-64==12.1.0=h3bb4806_10"), MatchSpec("gfortran_linux-64==12.1.0=h7ec3724_10")}
# Last 100 lines of the build log.
